src/libsystemd/sd-netlink/test-netlink.c

```diff
@@ -137,18 +137,18 @@ static void test_address_get(sd_netlink *rtnl, int ifindex) {
         assert_se(sd_netlink_message_read_string(r, IFA_LABEL, &label) == 0);
         assert_se(sd_netlink_message_read_cache_info(r, IFA_CACHEINFO, &cache) == 0);
 
-        assert_se((m = sd_netlink_message_unref(m)) == NULL);
+        assert_se((n = sd_netlink_message_unref(m)) == NULL);
         assert_se((r = sd_netlink_message_unref(r)) == NULL);
 
 }
 
 static void test_route(void) {
-        _cleanup_netlink_message_unref_ sd_netlink_message *req;
+        _cleanup_netlink_message_unref_ sd_netlink_message *req1, *req2;
```

Comment:
All _cleanup_* annotated variables need to be initialized to something that will make the associated cleanup function bail out. In this case, NULL.